test/system/clients_test.rb

```diff
@@ -5,26 +5,47 @@ class ClientTest < ApplicationSystemTestCase
     @client = clients(:felipe)
   end
 
+  teardown do
+    Rails.cache.clear
```

Comment:
why do you need this?

Reply:
I forget to remove it after my experiment